.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
 	"files.eol": "\n",
 	"deepscan.enable": true,
 	"typescript.tsdk": "node_modules\\typescript\\lib",
-	"powershell.codeFormatting.addWhitespaceAroundPipe": true
+	"powershell.codeFormatting.addWhitespaceAroundPipe": true,
+	"i18next.i18nPaths": "d:/r/archid/evlyn/src/locales"
```

Comment:
Yes I also have `d:/r/archid/evlyn` on my system.

Jokes aside, please modify this to be relative. Pretty sure it already travereres from `${workspaceRoot}` but otherwise add that or something.